core/themes/bartik/color/color.inc

```diff
@@ -15,14 +15,16 @@ $info = array(
     'titleslogan' => t('Title and slogan'),
     'text' => t('Text color'),
     'link' => t('Link color'),
+    'menu' => 'Menu link',
+    'activemenu' => t('Active menu link color'),
   ),
   // Pre-defined color schemes.
   'schemes' => array(
     'default' => array(
-      'title' => t('Blue Lagoon (default)'),
+      'title' => t('Blue Drop (default)'),
```

Comment:
If the default color is black, let's not call this "Blue Drop".  How about just "Black (default)"